### ISSUE TITLE
Text editor

### ISSUE THREAD

**Issue by robgpeak** [robgpeak/CompLinks-xclub#19]
Parx wants to change and the tag line verbiage to, "...exchange for Comp Dollars", and also repeat the (100 Rewards Everywhere points = 1 Parx Casino® Comp Dollar) in that bottom section of the How it works page. (I can do this, but I make this change now, I am going to lose the “Join Now” button functionality from the bottom section, and I would rather not.

You mentioned something about https://docs.ckeditor.com/ckeditor4/latest/guide/dev_framed.html and I wanted to find out where we were with that?

Even if you do get that editor in there, I would need the short code for the button...

**Comment by cgladue** [comment]
working on it now, you should be able to soon



Chad Gladue
*DevNull Technology, LLC*

**Comment by cgladue** [comment]
actually the HTML editor refuses to show the <md-button> tag its killing me, so i just removed the HTML editor for now, all your gonna do is bold and paragraphs anyways so doing it manually in the HTML should be fine for now... i can circle back again in the future to try and see a better way :(

**Comment by cgladue** [comment]
@robgpeak now on the HowItWorks page, i have made it work where it keeps the button, you dont need to use the strange |tag| anymore.

**Comment by robgpeak** [comment]
cool.  But I think I messed something up.  Question, if I click on the <> icon, I can see the html source, but I can't edit it.  Is that by design?  Is it possible that I can edit it and save it?  That would be good.

So for kicks I put another button in there just to see what would happen and then I removed it from the text editor, but it's still showing up in the source code and I can't remove it.  Have a look at https://shop.complinks.co/how-it-works in the first section, and then see the image below.

<img width="733" alt="screen shot 2018-03-22 at 9 26 41 pm" src="https://user-images.githubusercontent.com/28393235/37806767-1635b4a2-2e18-11e8-932d-1f5101897c8b.png">
